### PR TITLE
Fix a probable typo in changelog (`ISet` -> `IUniv`)

### PR DESCRIPTION
**Pull request by ice1000**
@ecavallo is it correct?

### CODE REVIEW

**Comment by ice1000** [comment]
I think most likely as ea34f611b111025016f0684dd8d2413edfbd1c7d

File: CHANGELOG.md [hunk 1]
```diff
@@ -63,8 +63,8 @@ Language
   (see [#5427](https://github.com/agda/agda/issues/5427)).
 
 * The cubical interval `I` now belongs to its own sort, `IUniv`, rather
-  than `SSet`. For `J : ISet` and `A : J → Set l`, we have
-  `(j : J) → A : Set l`, that is, the type of functions from a type in `ISet`
+  than `SSet`. For `J : IUniv` and `A : J → Set l`, we have
+  `(j : J) → A : Set l`, that is, the type of functions from a type in `IUniv`
```
Comment (plt-amy) [inline CHANGELOG.md]: `(j : J) → A j`

Comment (ice1000) [reply CHANGELOG.md]: Nice catch

Comment (ice1000) [reply CHANGELOG.md]: Fixed

**Pull request merged**